spec/features/hiring_staff_can_share_their_vacancy_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.feature 'Hiring staff can share their vacancy' do
+  let(:school) { create(:school) }
+  before(:each) do
+    stub_hiring_staff_auth(urn: school.urn)
+  end
+
+  scenario 'A school can visit their page as the job seeker would' do
+    vacancy = create(:vacancy, school: school)
+
+    visit school_path
+
+    click_on(vacancy.job_title)
+
+    click_on(I18n.t('jobs.view_public_link'))
+
```

Comment:
Can we remove the empty lines between the actions? I think setup - space - actions - space - expectation reads quite well so we dont really need the new lines.

Reply:
Done.